src/annotator/components/AdderToolbar.js

```diff
@@ -82,17 +137,24 @@ export default function AdderToolbar({
   return (
     <div
       className={classnames(
-        'hyp-u-border hyp-u-bg-color--white',
         'AdderToolbar',
+        'absolute select-none bg-white rounded shadow-adderToolbar',
+        // Because `.AdderToolbar` rules reset `all:initial`, we cannot use
+        // default border values from Tailwind and have to be explicit about
+        // all border attributes
```

Comment:
Attn.

Reply:
I will definitely look into this.

Reply:
@robertknight Do you think this rule could be added to the `:host` selector as well? Or are we keen to keep it "scoped" to just the adder toolbar?

Reply:
It probably would be a good idea to apply it to all of our shadow roots. I haven't tried to find out whether any other Hypothesis UI elements could be broken in unexpected ways by inherited styles, but there probably are some.

Reply:
Just a note that I went _real deep_ on this this afternoon and do not yet have a proposed solution. It's a tricksy one to do with inheritance and specificity and the specific structure of how this component (and other annotator components) are generated and added to their respective shadow DOMs. Learned a lot. Stuff to share at the next front-end sync. Ecosystem at an awkward teenager state WRT this stuff. I'll update some commenting but am not proposing to make any big moves in this particular changeset.